doc/interoperability/pandas.rst

```diff
@@ -0,0 +1,288 @@
+
+.. _pandas:
+
+=======================
+Pandas Interoperability
+=======================
+
+This part of the User Guide aims to highlight some of the particularities that
+arise when using Pandas datastructures as input for Scikit-learn.
+
+The basics of using Pandas and Scikit-learn
+==================================================================
+
+Scikit-learn supports the use of
+`Pandas DataFrames <http://pandas.pydata.org/pandas-docs/stable/>`__
+implicitly. However, this implicit support has its conditions and potential
+pitfalls and some (not all) of these will be outlined below.
+
+Reasons for these conditions and the difficulty of supporting Pandas, stems
+from the fact that Pandas data structures such as Series and DataFrames can
+hold heterogenous datatypes (different columns can contain different
+datatypes). Therefore, Series and DataFrames can be thought of as containers
```

Comment:
"while numpy arrays are designed to work with homogeneous data type, and do not support some rich dtypes available in Pandas, such as..." would summarise the remainder of the paragraph.